src/containers/NewPositionWrapper/NewPositionWrapper.tsx

```diff
@@ -375,7 +375,7 @@ export const NewPositionWrapper: React.FC<IProps> = ({
     if (id.length) {
       setPriceALoading(true)
       getCoingeckoTokenPrice(id)
-        .then(data => setTokenAPriceData(data))
+        .then(data => setTokenAPriceData({ price: data ?? 0 }))
```

Comment:
"no data" should be left if the fetch failed case

Reply:
if the price is 0, it results in "no data" being displayed